src/modules/tests/test-summary/cat-a-mod1/test-summary.cat-a-mod1.reducer.ts

```diff
@@ -0,0 +1,76 @@
+import { TestSummary } from '@dvsa/mes-test-schema/categories/AM1';
+import { createFeatureSelector } from '@ngrx/store';
+import * as fromMod1TestSummaryActions from './test-summary.cat-a-mod1.actions';
+import * as fromTestSummaryActions from '../common/test-summary.actions';
+
+export const initialState : TestSummary = {
+  routeNumber: null,
```

Comment:
this can be defaulted to the number 88 and stay that way as it won't be changed..